code/game/mecha/equipment/tools/work_tools.dm

```diff
@@ -34,6 +34,14 @@
 		return
 	if(isobj(target))
 		var/obj/O = target
+		if(istype(O, /obj/machinery/door/firedoor))
+			var/obj/machinery/door/firedoor/D = O
+			D.try_to_crowbar(src,chassis.occupant)
+			return
+		if(istype(O, /obj/machinery/door/airlock/))
+			var/obj/machinery/door/airlock/D = O
+			D.try_to_crowbar(src,chassis.occupant)
+			return
```

Comment:
Can't he just check for /obj/machinery/door instead of having two separate typechecks?

Reply:
lol